### PR TITLE
Add birthday keyword and new image to csv import

### PR DESCRIPTION
**Pull request by Ojself**
DO NOT MERGE before pull request [1069](https://github.com/schul-cloud/schulcloud-server/pull/1069) is merged to server.

csv
(please ignore the thr theme)

## Allgemein
- [X] Links zu verwandten PRs anderer Repositories
  - https://github.com/schul-cloud/schulcloud-server/pull/1069
- [X] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-3607

## Dokumentation
- [X] Dokumentation (wenn notwendig)
  - Code
  - Confluence - https://docs.schul-cloud.org
  - Release Notes - wenn es sich um große Feature-Releases handelt
- [X] mind. 1 Screenshot bei Content-Änderungen
- [X] Changelog-Eintrag

### CODE REVIEW

File: views/administration/forms/form-import.hbs [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <a href="https://docs.schul-cloud.org/x/NYBgAw" target="_blank">hier</a>.
 </p>
 <p>
-    <img src="/images/csv_example.png" />
+    <img src="/images/csv_example2.png" />
```
Comment (adrianjost) [inline views/administration/forms/form-import.hbs]: why have you attached a `2` to the filename? Seems unnecessary to me.

File: views/administration/forms/form-import.hbs [hunk 2]
```diff
@@ -31,6 +32,14 @@
...
-        Alternativ kannst du die Registrierungslinks später abrufen und deinen Nutzern per Mail oder QR-Druckbogen zukommen lassen.
+        Alternativ kannst du die Registrierungslinks später abrufen und deinen Nutzern per Mail oder QR-Druckbogen
+        zukommen lassen.
     </p>
-</div>
+    <p class="lead">
+        ⚠️ ACHTUNG: Beim Import großer Datensätze kommt es kurz nach dem Start zu einer Fehlermeldung. Im Hintergrund
+        läuft der Importprozess trotzdem unsichtbar weiter. Zeitdauer: circa 4 Minuten pro 1000 Nutzer:innen. Bitte
+        warte die entsprechende Zeit ab und aktualisiere dann dein Browserfenster, um die aktualisierte
+        Schüler:innen-Übersicht überprüfen zu können.
```
Comment (adrianjost) [inline views/administration/forms/form-import.hbs]: Maybe we sould add a hint, that we are "working" on this problem?
@dominikbruechner @konstantin-sc @sarahschaefer

**Comment by konstantin-sc** [comment]
@adrianjost would be ok for me without hint. but a "derzeit" before "kurz" should do the job

File: views/administration/forms/form-import.hbs (hunk 1 above)
Comment (Ojself) [reply views/administration/forms/form-import.hbs]: fixed.

File: views/administration/forms/form-import.hbs (hunk 2 above)
Comment (Ojself) [reply views/administration/forms/form-import.hbs]: fixed,
the line is now:
[...] derzeit es kurz nach

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5251601&bid=17208931)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5251601&bid=17208931)

**Comment by Parnswir** [comment]
@adrianjost please re-review

**Pull request merged**